src/main/resources/db/migration/V3_0__add_sessions.sql

```diff
@@ -0,0 +1,27 @@
+create table keyworker_session
+(
+    id                uuid       not null primary key,
+    occurred_at       timestamp  not null,
+    person_identifier varchar(7),
+    staff_id          bigint     not null,
```

Comment:
We should take the username too. I still want to transition away from staff ids at some point and the usernames from the case notes may help with that